release-02/code/config.ru

```diff
@@ -4,5 +4,7 @@ require 'dotenv'
 Dotenv.load
 
 require './minitwit'
+require './minitwit_sim_api'
 
-run MiniTwit::App.freeze.app
+#run MiniTwit::App.freeze.app
```

Comment:
Why is the main app no being run anymore? This is the config file for the app and the API. Shouldn't both be started?